docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1612,4 +1612,4 @@ Hive 3-related limitations
   Trino returns different results than Hive.
 
 * Trino does not support gathering table statistics for Hive transactional tables.
-  You must use Hive to gather table statistics with ``ANALYZE TABLE COMPUTE STATISTICS`` after table creation.
\ No newline at end of file
+  You must use Hive to gather table statistics with ``ANALYZE TABLE COMPUTE STATISTICS`` after table creation.
```

Comment:
pre-existing, but since you're changing this line.... see https://github.com/trinodb/trino/pull/15637#discussion_r1123367663

Reply:
Sent #16376